src/app/home/home.component.less

```diff
@@ -1,14 +1,19 @@
 @import '../../global/style/global';
-
+.custom-style {
```

Comment:
this should be in the host selector, preferably not with the .custom-style rules